extensions/anonymized-location/examples/item.json

```diff
@@ -0,0 +1,241 @@
+{
+    "stac_version": "1.0.0-beta.1",
+    "stac_extensions": [
+      "eo",
+      "view",
+      "anon"
+    ],
+    "id": "LC08_L1TP_107018_20181001_20181001_01_RT",
+    "collection": "landsat-8-l1",
+    "type": "Feature",
+    "bbox": [
+        148,
+        59,
+        152,
+        60
+    ],
+    "geometry": {
+        "type": "Polygon",
+        "coordinates": [
+            [
+                [
+                    152,
+                    60
+                ],
+                [
+                    148,
+                    59
+                ],
+                [
+                    148,
+                    60
+                ],
+                [
+                    152,
+                    59
+                ],
+                [
+                    152,
+                    60
+                ]
+            ]
+        ]
+    },
+    "properties": {
+        "platform": "landsat-8",
+        "instrument": ["oli", "tirs"],
+        "datetime": "2018-10-01T01:08:32.033Z",
+        "gsd": 30,
+        "view:sun_azimuth": 168.8989761,
+        "view:sun_elevation": 26.32596431,
+        "view:off_nadir": 0,
+        "landsat:path": 107,
+        "landsat:row": 18,
+        "eo:cloud_cover": 78,
+        "anon:precision": 1
+    },
+    "assets": {
+        "ANG": {
+            "href": "https://landsat-pds.s3.amazonaws.com/c1/L8/107/018/LC08_L1TP_107018_20181001_20181001_01_RT/LC08_L1TP_107018_20181001_20181001_01_RT_ANG.txt",
+            "title": "Angle coefficients file",
+            "type": "text/plain"
+        },
+        "B1": {
+            "href": "https://landsat-pds.s3.amazonaws.com/c1/L8/107/018/LC08_L1TP_107018_20181001_20181001_01_RT/LC08_L1TP_107018_20181001_20181001_01_RT_B1.TIF",
```

Comment:
This example confuses me a bit, since landsat data from the public bucket pretty easily lets you get its location. I think it'd be better to use a more 'real' example of some image that is commercial / private.

Reply:
Like the assets linked should probably not be a geotiff, since that would have the actual location in it.